Makefile

```diff
@@ -173,7 +173,7 @@ dapr.yaml: check-docker-env
 	$(info Generating helm manifest $(HELM_MANIFEST_FILE)...)
 	@mkdir -p $(HELM_OUT_DIR)
 	$(HELM) template \
-		--include-crds=true --set dapr_config.dapr_config_chart_included=false --set-string global.tag=$(DAPR_TAG) --set-string global.registry=$(DAPR_REGISTRY) $(HELM_CHART_DIR) > $(HELM_MANIFEST_FILE)
+		--include-crds=true --set dapr_config.dapr_config_chart_included=false --set-string global.tag=$(REL_VERSION) --set-string global.registry=$(DAPR_REGISTRY) $(HELM_CHART_DIR) > $(HELM_MANIFEST_FILE)
```

Comment:
will this break running e2e tests locally?

Reply:
I don't think so? But, good question, I'll run them right now. Might need to update the docs.

Reply:
Yeah definitely need to tweak some things. Good catch.